docs/app/pages/components/components-sections/tables/filters/filters.component.html

```diff
@@ -58,8 +58,8 @@
     <tr uxd-api-property name="clearTooltip" type="string" defaultValue="Clear All">
         Defines the text to display on the clear button tooltip.
     </tr>
-    <tr uxd-api-property name="clearAll" type="TemplateRef || string">
-        Overwrite the clear all button with a string.
+    <tr uxd-api-property name="clearAllTemplate" type="TemplateRef || string">
```

Comment:
With the latest changes, this is now added as a child, so it probably is not correct to document this as an input anymore as this could confuse people.

Reply:
Yes, for now let's just use the same documentation format as the templates in the calendar components.